docs/modules/ROOT/pages/tutorial-kubernetes.adoc

```diff
@@ -0,0 +1,361 @@
+
+We follow the https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx[KIND documentation^] to deploy Nginx.
+
+[source,console]
+----
+$ INGRESS_VERSION=$(curl https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/stable.txt)
+$ kubectl apply -f "https://raw.githubusercontent.com/kubernetes/ingress-nginx/${INGRESS_VERSION}/deploy/static/provider/kind/deploy.yaml"
+$ kubectl wait --namespace ingress-nginx \
+  --for=condition=ready pod \
+  --selector=app.kubernetes.io/component=controller \
+  --timeout=120s
+$ kubectl apply -f ingress.yaml
+----
+
+The last step deploys the actual `Ingress` configuration as below.
+
+[source,yaml]
+----
+---
+# Ingress to forward to our web server including sub-paths: we should just forward what we need but for local testing just sending it all.
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+    name: couchbase-ingress
+    annotations:
+        kubernetes.io/ingress.class: nginx
+        nginx.ingress.kubernetes.io/rewrite-target: /$1$2
+        nginx.ingress.kubernetes.io/use-regex: 'true'
+spec:
+    rules:
+        - http:
+              paths:
+                  - path: /(.*)(/|$)
+                    pathType: Prefix
+                    backend:
+                        service:
+                            name: couchbase-grafana-http
+                            port:
+                                number: 8080
+---
+----
+
+== Using CMOS
+
+Once this is all deployed, which can be automated using a https://github.com/couchbaselabs/observability/blob/main/examples/kubernetes/run.sh[fully scripted version^], then you should be able to browse to http://localhost (or wherever you are deploying it with the appropriate `Ingress`).
+
+You should then hit the landing page which includes links to this documentation.
+
+You can verify all the Prometheus targets are set up.
+
+.Successful Prometheus discovery
+image::prometheus-successful-targets.png[]
+
+Whilst the pods are coming up, some may report as failing but these will resolve once the pods are running.
+
+.Failing targets during startup
+image::prometheus-failing-targets.png[]
+
+=== CMOS cluster configuration
+
+[NOTE]
+====
+Unfortunately cluster monitor is currently designed for an on-premise solution so will not cope with auto-discovery on Kubernetes yet.
+Without this the various Grafana dashboards will not function correctly so you have to do it manually for now.
+====
+
+Kubernetes support is on the roadmap but, in the meantime, to manually add the cluster details you will need the credentials for accessing the REST API and a pod name for one of the Couchbase Server pods in the cluster (it will bootstrap the rest).
+
+These can then be added via the `Add Cluster` link on the landing page.
+Make sure to disable the `Add Nodes to Prometheus` option as this is handled already.
```

Comment:
Hm, I think I may have removed this in the latest iteration of the form - sounds like something we'll want to have optional (even if enabled by default).

Reply:
Yeah we do, it still seems to work for me but maybe it is adding an unscrapeable target?

Reply:
It'll be adding it as a separate target (check localhost:8080/prometheus/targets, you'll see something like `couchbase-server-managed-1`).

Reply:
Filed https://issues.couchbase.com/browse/CMOS-156